src/Core/src/Credentials/IniFileProvider.php

```diff
@@ -93,14 +93,24 @@ private function getCredentialsFromProfile(array $profilesData, string $profile,
             return $this->getCredentialsFromRole($profilesData, $profileData, $profile, $circularCollector);
         }
 
+        if (isset($profileData[IniFileLoader::KEY_SSO_SESSION])) {
+            if (!class_exists(SsoClient::class)) {
```

Comment:
shouldn't this check for `SsoOidcClient` instead ?

Reply:
already addressed by https://github.com/async-aws/aws/pull/1833